app/client/build.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh
 
 set -e
+if [ -z $GIT_SHA ]
```

Comment:
Please use bash + double braces + unquoted variables, or sh + quoted variables + single braces. Only those two combinations work reliably.

```suggestion
if [ -z "$GIT_SHA" ]
```